packages/devtools_test/lib/src/mocks/fake_service_manager.dart

```diff
@@ -11,10 +11,7 @@ import 'package:flutter/foundation.dart';
 import 'package:mockito/mockito.dart';
 import 'package:vm_service/vm_service.dart';
 
-import 'fake_service_extension_manager.dart';
-import 'fake_vm_service_wrapper.dart';
-import 'generated.mocks.dart';
-import 'mocks.dart';
+import '../../devtools_test.dart';
```

Comment:
switch back to importing these files directly instead of importing the files that exports them

Reply:
done

Reply:
I'm not seeing the commit for this?

Reply:
now for sure